src/drt/src/db/obj/frInst.cpp

```diff
@@ -35,77 +35,28 @@ namespace drt {
 Rect frInst::getBBox() const
 {
   Rect box = getMaster()->getBBox();
-  dbTransform xform = getTransform();
-  Point s(box.xMax(), box.yMax());
-  updateXform(xform, s);
+  dbTransform xform = getDBTransform();
   xform.apply(box);
   return box;
 }
 
 Rect frInst::getBoundaryBBox() const
 {
   Rect box = getMaster()->getDieBox();
-  dbTransform xform = getTransform();
-  Point s(box.xMax(), box.yMax());
-  updateXform(xform, s);
+  dbTransform xform = getDBTransform();
   xform.apply(box);
   return box;
 }
 
-dbTransform frInst::getUpdatedXform(bool noOrient) const
+#include <iostream>
```

Comment:
Should be at the top of the file with the other includes

Reply:
Oh sorry, I used iostream for debugging, I missed it when preparing the PR